crates/modelardb_common/src/parser.rs

```diff
@@ -742,11 +737,28 @@ pub fn parse_sql_expression(df_schema: &DFSchema, sql_expr: &str) -> Result<DFEx
 /// which was released under version 2.0 of the Apache License.
 ///
 /// [rewrite_expr.rs]: https://github.com/apache/arrow-datafusion/blob/main/datafusion-examples/examples/rewrite_expr.rs
-struct EmptyContextProvider {
+struct ParserContextProvider {
```

Comment:
It might not be necessary but maybe change the doc comment for this struct slightly since it has been changed.

Reply:
Fixed.